### PR TITLE
DOCKER DOCKER DOCKER DOCKER

### PR DESCRIPTION
**Pull request by keynmol**
## WIP

A little tool that samples Wikipedia XML dump. Outputs nice progress to `stderr`

```
docker run -i -v /<path-to-wiki-xml>:/data idio/wikistats-split-wiki-dump /data/enwiki-20150403-small.xml 1000 > output.xml
```
And output.xml will contain 1000 purely random articles from the original wiki dump.

### CODE REVIEW

File: split-wikipedia-xml/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:14.04
+
+RUN apt-get install python3.4
+RUN apt-get install perl
```
Comment (muz) [inline split-wikipedia-xml/Dockerfile]: I'd recommend single lining these `apt` packages down, and munge through a `-y` flag.

`RUN apt-get install -y python3.4 perl`

File: split-wikipedia-xml/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+RUN locale-gen en_US en_US.UTF-8
+RUN echo "export LANG=en_US.UTF-8" >> /root/.bashrc
+
+RUN mkdir /work
+ADD splitter.sh /work/splitter.sh
```
Comment (muz) [inline split-wikipedia-xml/Dockerfile]: Prefer `COPY` over `ADD`. It's simpler, and doesn't bring into question any manner of magic that `ADD` offers.

**Comment by 36degrees** [comment]
Suuuure why not.

**Comment by mal** [comment]
I know this pre-dates the introduction of waffle, but it'd be good to connect it to an issue. If you do, please remove the `review` label from this PR and add it to the issue, thanks! :grinning:

**Pull request closed without merging**